HTTPServerProject.Tests/ResponseTests.cs

```diff
@@ -22,7 +22,7 @@ public void AddAndGetHeaderTest()
     {
         var expected = new List<string>() { "HTTP/1.1 200 OK", "Location: Earth", "", "Hello World!" };
         var writer = new TestWriteStreams(eStream);
-        var headers = new List<string>(){"Location: Earth"};
+        var headers = new List<string>() { "Location: Earth" };
```

Comment:
The commit here doesn't seem to match the actual code. Any idea what's going on here?

Reply:
Ah I see what happened - some commit message got re-used by mistake. This often happens in IntellJ because it has a (annoying IMO) feature of keeping the commit message from the last time you committed in the dialog. Make sure you double check commit messages when you commit.

Reply:
Yes, my mistake. That was one of my first commits using Rider and I double tapped the commit message.